test/functional/cursorstream.test.js

```diff
@@ -207,7 +207,8 @@ describe('Cursor Streams', function () {
     }
   });
 
-  it('should stream documents across getMore command and count correctly', {
+  // Fails on MacOS on Evergreen (NODE-3713)
```

Comment:
Needs unskip ticket

Reply:
Added todo comment to https://jira.mongodb.org/browse/NODE-3819